WrightTools/data/_channel.py

```diff
@@ -91,17 +93,6 @@ def null(self):
             self.attrs['null'] = 0
         return self.attrs['null']
 
-    @property
-    def info(self):
```

Comment:
Any particular reason to remove the info dictionary?

Reply:
yeah, I just wasn't seeing it get used

do you think it's useful?